gulpfile.coffee

```diff
@@ -43,10 +43,10 @@ getRPath = (path) ->
   return path
 
 getLegacyRPath = (path) ->
-  console.warn  "Warning: All R code in modules path 'modules/**/src/server/**/*.{R,r}' should be moved to 'modules/**/src/server/r/*.{R,r}'"
+  # console.warn  "Warning: All R code in modules path 'modules/**/src/server/**/*.{R,r}' should be moved to 'modules/**/src/server/r/*.{R,r}'"
```

Comment:
I don't think these warnings should be commented out. The point of them is to spur people to move files in modules in the legacy path format into the updated path format. If all the R files are put in the right place, we won't see these warnings on compile.